### PR TITLE
Lg 2420/signup marketing page

### PR DESCRIPTION
**Pull request by ovcOS**
The only `signup` content available so far has been named `main` in Contentful. So, to see the page rendered go to `.../signup/main`

### CODE REVIEW

File: src/components/forms/Fields.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        setState(e.target.value);
+        setFormStatus(FORM_STATUSES.IDLE);
+      }}
+      value={state}
+      name={name}
+    />
+  </div>
+);
+
+export const InvalidFieldHints = ({ formStatus }: { formStatus: string }) => {
```
Comment (colmarius) [inline src/components/forms/Fields.jsx:40]: I see you have added a `FormStatus` type, should we have`formStatus: string` -> `formStatus: FormStatus`?

File: src/components/forms/lib/handleSignupForm.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+const redirectToApp = ({ name, email }: { name: string, email: string }) => {
+  const data = encodeFormData({ name, email });
+  const url = `${appUrl}/signup/${data}?utm_source=landing_signup`;
+  if (window) {
+    window.location = url;
+  }
+};
+
+export const handleSignupForm = ({
```
Comment (colmarius) [inline src/components/forms/lib/handleSignupForm.js:20]: suggestion: we could add `Submit` suffix, as in `handleSignupFormSubmit`

**Review by colmarius — COMMENTED** [review]
Reviewed and left some small feedback.

In addition when error message shows up to fill require fields, it is not clear what are the required fields (*). For example:

Screenshot 2020-09-03 at 15 33 14

We could fix this issue by adding * to the name / email placeholders, like so:

Screenshot 2020-09-03 at 15 33 40

Tested and functionality works fine 👍. Nice job.

File: src/components/forms/Fields.jsx (hunk 1 above)
Comment (ovcOS) [reply src/components/forms/Fields.jsx:40]: Absolutely, missed this one

**Review by colmarius — APPROVED** [review]
Tested once more, last changes look good :-)

**Pull request merged**